--- /home/rick/nas_rram/ofa/once-for-all/ofa/imagenet_classification/networks/resnets.py
+++ /home/rick/nas_rram/ofa/once-for-all/ofa/imagenet_classification/networks/resnets.py
@@ -5,8 +5,7 @@
 	# BASE_DEPTH_LIST = [2, 2, 2, 2]
 	BASE_DEPTH_LIST = [2, 2, 2, 2]
 	# STAGE_WIDTH_LIST = [64, 128, 256, 512]
-	STAGE_WIDTH_LIST = [16, 32, 64, 128]
-	# STAGE_WIDTH_LIST = [2, 4, 8, 16]
+	STAGE_WIDTH_LIST = [2, 4, 8, 16]
 
 	def __init__(self, input_stem, blocks, classifier):
 		super(ResNets, self).__init__()